README.md

```diff
@@ -52,4 +52,21 @@ can be overriden by the environment variable `DOCKER_GATEWAY_HOST`:
 ```bash
 EXPORT DOCKER_GATEWAY_HOST=`docker network inspect --format='{{range .IPAM.Config}}{{.Gateway}}{{end}}'
 npm start
+```
+
+## Debugging locally
+
+To save the arguments (`config`, `scdata`, etc) to a task function, specify DEBUG_STEP and DEBUG_PATH:
+
+```bash
+# e.g. DEBUG_STEP=dataIntegration
+DEBUG_STEP=task_name DEBUG_PATH=/path/to/debug/folder npm start
+```
+
+When the pipeline is run, it will save the arguments to the specified `task_name` in `DEBUG_PATH`. You
+can load these into your R environment:
+
+```R
+# clicking the file in RStudio does this for you
+load('/path/to/debug/folder/{task_name}_{sample_id}.RData')
```

Comment:
In order to load in multiple samples, this trick might be worth mentioning to create an enviornment for each (load would otherwise overwrite the previous objects)
So something like this could be added below this line:
     `load('/path/to/debug/folder/{task_name}_{sample_id}.RData', envir = task_name <- new.env())`

Reply:
Thanks - added to example